linters/renovate/plugin.yaml

```diff
@@ -1,4 +1,11 @@
 version: 0.1
+tools:
+  definitions:
```

Comment:
Can this linter not be refactored to use the tool?

Reply:
no, it can't - due to the symlinks. can follow up to support that functionality with tools.